style.css

```diff
@@ -608,3 +615,186 @@ textarea {
+    background-image: url(/images/desktop/desktopBg.svg);
+    background-size: 90%;
+    background-position: 0% 15%;
+    align-items: flex-start;
+    justify-content: center;
+    padding-left: 140px;
+    padding-bottom: 150px;
+    border-bottom-left-radius: 165px;
+  }
+
+  .headerOne {
+    width: 600px;
+    line-height: 60px;
+    justify-content: start;
+  }
+
+  .desc {
+    width: 85%;
+    line-height: 28px;
+    font-size: 20px;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .socials {
+    width: 30%;
+    margin-right: 10px;
+  }
+
+  .socialsM {
+    width: 30%;
+    margin-right: 10px;
+    padding-top: 6px;
+  }
+
+  .project {
+    padding: 20px 16px 24px 16px;
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    border-radius: 16px;
+    border: 1px solid #dfe1e6;
+    background-color: #fff;
+    width: auto;
+    height: auto;
+    justify-content: space-around;
+  }
+
+  .description {
+    padding: 48px;
+    margin-top: 32px;
+  }
+
+  .viewProject {
+    width: auto;
+    height: auto;
+  }
+
+  .project img {
+    height: 50%;
+    width: 50%;
+  }
+
+  .subTitle {
+    font-weight: bold;
+    font-size: 16px;
+    line-height: 24px;
+    margin-left: 0;
+  }
+
+  .info {
+    margin-left: -10px;
+  }
+
+  .projects .projectsReverse {
+    width: auto !important;
+    row-gap: 0;
+  }
+
+  .aboutContainer {
+    flex-direction: row;
+    border-top-right-radius: 80px;
+    background-color: #fff;
+  }
+
+  .aboutMyself {
+    width: 40%;
+    margin-top: -110px;
+    padding: 48px;
+    margin-left: 100px;
+  }
+
+  .about {
+    margin-left: 18px;
+    width: 50%;
+    height: 16%;
+  }
+
+  .mySkills {
+    width: 60%;
+  }
+
+  .skillsTitle {
+    width: 80%;
+  }
+
+  .skillsDescList {
+    width: 100px;
+    height: 100px;
+    margin-left: 0;
+    margin-right: 10px;
+    padding: 20px;
+    flex-direction: column;
+  }
+
+  .skillsDesc {
+    flex-direction: row;
+    align-items: flex-start;
+    justify-content: flex-start;
+    border-radius: 8px;
+    margin-left: -77px;
+    padding: 9px 30px 0 80px;
+  }
+
+  .contact {
+    background-image: url('images/desktop/contact-desktop.svg');
+    background-position: center center;
+    background-size: auto;
+  }
+
+  .contactContainer {
+    width: 25%;
+  }
+
+  .navDesk {
+    display: flex;
+    width: 50%;
+    margin-left: 50%;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .menuDesk {
+    display: inline;
+  }
+
+  .menuDesk a {
+    font-family: Poppins, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    list-style: none;
+    margin-right: 16px;
+    text-decoration: none;
+  }
+
+  .formSubmit {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .descriptionReverse {
+    order: -1;
+    padding: 24px;
+    margin-top: 32px;
+  }
+}
```

Comment:
- [ ] [REQUIRED] Great desktop design 👍🏽 . Looks almost like the one provided in the template, however, your mobile versions have some issues you need to put in place. The cards are no more at the center of the page. Please fix this to get the project approved.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![1](https://user-images.githubusercontent.com/58692590/154468950-ef149492-e292-4ae1-b433-127e9604dda9.png) | ![11](https://user-images.githubusercontent.com/58692590/154468980-096ee609-4462-480c-9ea9-d8664313cf6e.png)

Reply:
Thank you very much for pointing this out! I didn't even realize that I forgot to add the padding back after I got rid of the bottom scroll bar!

Reply:
You are welcome!